modules/Bio/EnsEMBL/Production/Pipeline/Xrefs/gencode_sources.json

```diff
@@ -105,42 +105,50 @@
     {
       "name" : "RefSeq_dna",
       "parser" : "RefSeqGPFFParser",
-      "file" : "ftp://ftp.ncbi.nih.gov/refseq/H_sapiens/mRNA_Prot/*rna.gbff.gz",
+      "file" : "https://ftp.ncbi.nih.gov/refseq/H_sapiens/mRNA_Prot/*rna.gbff.gz",
       "method" : "--bestn 5",
       "query_cutoff" : 90,
       "target_cutoff" : 90,
-      "release" : "ftp://ftp.ncbi.nih.gov/refseq/release/release-notes/RefSeq-release*.txt",
-      "priority" : 2
+      "release" : "https://ftp.ncbi.nih.gov/refseq/release/release-notes/RefSeq-release*.txt",
```

Comment:
Just a remark, I believe the code is actually using the FTP protocol to download the file.

Reply:
Yes the code uses ftp protocol if the link is an ftp one. However, since ftp is no longer supported, we've decided to change this to https (which is available) for more stability.

Reply:
What do you mean exactly by `supported` ? 
![image](https://user-images.githubusercontent.com/611039/198010517-da6af70a-1d8b-4253-8bca-c3dc4016a307.png)